### ISSUE TITLE
Ankündigung über nächste Runde

### ISSUE THREAD

**Issue by GeneralMine** [brean/discord-bot-cud#8]
Den Bot an ein Text Channel binden und Xsec vor Rundenwechsel ankündigen, dass die Runde in Xsec endet.

Ich empfehle X=15sec

- Request eines Teilnehmers

**Comment by brean** [comment]
Aktuell antwortet der Bot auf dem gleichen channel in dem .shuffle aufgerufen wurde, aber wir könnten ihm beibringen einen Announcement-Channel einzufügen, der könnte dann auch für generelle Timer (z.B. 10 Minuten für die Ideenfindung) benachrichtigen á la "Hey bot, setze einen Timer auf 10 Minuten" :-)

**Comment by GeneralMine** [comment]
Genau, reicht dafür nicht auch der announcements channel den wir schon haben?
Wir dann vllt bisschen gespame, aber es wäre klarer und wir können im Vorhinein sagen "öffnet jetzt den Channel"

**Comment by brean** [comment]
ich würde es als Parameter hinzufügen, dann könnte man z.B. den announcement-channel angeben.
